heroic/hc_coven_of_shivarra.simc

```diff
@@ -1,5 +1,33 @@
 # movement
+# Whirling Saber
+raid_events+=/movement,cooldown=30,duration=1,first=10,last=200,players_only=1,player_chance=0.2
+raid_events+=/movement,cooldown=30,duration=1,first=400,players_only=1,player_chance=0.2
+# Storm of Darkness
+# could spawn one under you, so not totally 100%
+raid_events+=/movement,cooldown=60,duration=3,first=30,last=330,players_only=1,player_chance=0.8
```

Comment:
There is a slight chance that you will actually get storm to spawn under you so you don't need to move.

Reply:
I would add some stddev for variance in duration, as you can be far/close to one.